GLMakie/src/GLVisualize/visualize/particles.jl

```diff
@@ -323,6 +346,22 @@ function sprites(p, s, data)
         # to offset.
         data[:offset] = map(rescale_glyph, p[1], font, offset)
         data[:scale] = map(rescale_glyph, p[1], font, scale)
+    
+    elseif to_value(p[1]) isa BezierPath
+        scale = map(combine_scales,
+            pop!(data, :scale, Observable(nothing)),
+            pop!(data, :scale_x, Observable(nothing)),
+            pop!(data, :scale_y, Observable(nothing)),
+            pop!(data, :scale_z, Observable(nothing))
+        )
+        # TODO 
+        # marker_offset should be Vec2f(0) by default for BezierPaths
+        # once that is the case we should switch to this offset
+        # offset = get(data, :offset, Observable(Vec2f(0)))
+        offset = Observable(Vec2f(0))
```

Comment:
It would be good if BezierPath scatters would get some different defaults. `marker_offset` defaults to centering https://github.com/JuliaPlots/Makie.jl/blob/78d71c5609684ebbeaea7e073740582798d59330/src/interfaces.jl#L65-L68 which isn't what we want here and it might make sense to have `Markersize = 1` as the default?